wandb/sdk/launch/_project_spec.py

```diff
@@ -148,10 +159,13 @@ def get_entry_point(self, entry_point: str) -> "EntryPoint":
     def _fetch_project_local(self, internal_api: Api) -> None:
         """Fetch a project into a local directory, returning the path to the local project directory."""
         assert self.source != LaunchSource.LOCAL
+        _logger.info("Fetching project locally...")
         if utils._is_wandb_uri(self.uri):
+            _logger.info("Fetching run info...")
```

Comment:
this and line 168 seem to be duplicates?

Reply:
Ah good catch.